spec/src/constructorio.js

```diff
@@ -81,6 +81,27 @@ describe(`ConstructorIO${bundledDescriptionSuffix}`, () => {
     expect(instance.options).to.have.property('networkParameters').to.equal(networkParameters);
   });
 
+  it('Should return an instance with correct serviceUrl when a http serviceUrl is passed', () => {
```

Comment:
The description says `http serviceUrl is passed` but we are passing an `https` value on line 87. Can we update the test to have an `http` value there. 
Also it might be nice to drop the other parameters and make it more similar to [this test](https://github.com/Constructor-io/constructorio-node/pull/193/files#diff-eeecf5f88f630f2dbfd79b209470f3d4707cd0cb828861731b3f36887fb8f29cR56) so it's clear what's being tested here

Reply:
should be taken care of, thanks for pointing it out.